.doom.d/config.org
#+TITLE: Emacs Config
#+STARTUP: showeverything


$DOOMDIR/config.el -*- lexical-binding: t; -*-

 Place your private configuration here! Remember, you do not need to run 'doom
 sync' after modifying this file!

* Email
 Some functionality uses this to identify you, e.g. GPG configuration, email
 clients, file templates and snippets.
#+BEGIN_SRC emacs-lisp
(add-to-list 'load-path "/usr/local/share/emacs/site-lisp/mu4e")

#+END_SRC

* Fonts
 Doom exposes five (optional) variables for controlling fonts in Doom. Here
 are the three important ones:

 + `doom-font'
 + `doom-variable-pitch-font'
 + `doom-big-font' -- used for `doom-big-font-mode'; use this for
   presentations or streaming.

 They all accept either a font-spec, font string ("Input Mono-12"), or xlfd
 font string. You generally only need these two:
#+BEGIN_SRC emacs-lisp
(setq doom-font (font-spec :family "monospace" :size 14 :weight 'semi-light)
       doom-variable-pitch-font (font-spec :family "sans" :size 13))
#+END_SRC

* Theme

#+BEGIN_SRC emacs-lisp
(setq doom-theme 'doom-dracula)
#+END_SRC

* Directory

#+BEGIN_SRC emacs-lisp
(setq org-directory "~/org/")
#+END_SRC

* Line Settings

#+BEGIN_SRC emacs-lisp
(setq display-line-numbers-type t)
(global-set-key "\C-x\ t" 'toggle-truncate-lines)
#+END_SRC

* Browser(eww)

#+BEGIN_SRC emacs-lisp
(setq browse-url-browser-function 'eww-browse-url)
#+END_SRC

* Neotree

#+BEGIN_SRC emacs-lisp
(setq neo-window-fixed-size nil)
#+END_SRC

* Magit
#+BEGIN_SRC emacs-lisp
(require 'magit)
#+END_SRC
* Split

#+BEGIN_SRC emacs-lisp
(defun prefer-horizontal-split ()
  (set-variable 'split-height-threshold nil t)
  (set-variable 'split-width-threshold 40 t)) ; make this as low as needed
(add-hook 'markdown-mode-hook 'prefer-horizontal-split)
#+END_SRC

* File Manager(dired)
#+BEGIN_SRC emacs-lisp
(map!
  (:after dired
    (:map dired-mode-map
     "C-x i" 'peep-dired
     )))
(evil-define-key 'normal peep-dired-mode-map (kbd "j") 'peep-dired-next-file
                                             (kbd "k") 'peep-dired-prev-file)
(add-hook 'peep-dired-hook 'evil-normalize-keymaps)
#+END_SRC
* Elfeed
#+BEGIN_SRC emacs-lisp
(custom-set-variables
 '(elfeed-feeds
   (quote
    (("https://www.reddit.com/r/linux.rss" reddit linux)
     ("https://www.gamingonlinux.com/article_rss.php" gaming linux)
     ("https://hackaday.com/blog/feed/" hackaday linux)
     ("https://opensource.com/feed" opensource linux)
     ("https://linux.softpedia.com/backend.xml" softpedia linux)
     ("https://itsfoss.com/feed/" itsfoss linux)
     ("https://www.zdnet.com/topic/linux/rss.xml" zdnet linux)
     ("https://www.phoronix.com/rss.php" phoronix linux)
     ("http://feeds.feedburner.com/d0od" omgubuntu linux)
     ("https://www.computerworld.com/index.rss" computerworld linux)
     ("https://www.networkworld.com/category/linux/index.rss" networkworld linux)
     ("https://www.techrepublic.com/rssfeeds/topic/open-source/" techrepublic linux)
     ("https://betanews.com/feed" betanews linux)
     ("http://lxer.com/module/newswire/headlines.rss" lxer linux)
     ("https://distrowatch.com/news/dwd.xml" distrowatch linux))))
 '(package-selected-packages
   (quote
    (exwm peep-dired nav-flash evil-mu4e elfeed))))
#+END_SRC
 Here are some additional functions/macros that could help you configure Doom:

 `load!' for loading external *.el files relative to this one
 `use-package!' for configuring packages
 `after!' for running code after a package has loaded
 `add-load-path!' for adding directories to the `load-path', relative to
   this file. Emacs searches the `load-path' when you load packages with
   `require' or `use-package'.
 `map!' for binding new keys

 To get information about any of these functions/macros, move the cursor over
 the highlighted symbol at press 'K' (non-evil users must press 'C-c c k').
 This will open documentation for it, including demos of how they are used.

 You can also try 'gd' (or 'C-c c d') to jump to their definition and see how
 they are implemented.


.surf/history.txt
2020-12-27 15:14:52 https://lukesmith.xyz/ Luke Smith's Webpage
2020-12-27 15:16:28 http://suckless.com/ http://suckless.com/
2020-12-27 15:26:57 https://lukesmith.xyz/ Luke Smith's Webpage
2020-12-27 15:28:05 https://lukesmith.xyz/ Luke Smith's Webpage
2020-12-27 15:45:49 http://suckless.org/ software that sucks less | suckless.org software that sucks less
2020-12-27 15:46:25 https://www.wikipedia.org/ Wikipedia
2020-12-27 15:46:53 http://surf.suckless.org/ surf | suckless.org software that sucks less
2020-12-27 15:46:55 http://surf.suckless.org/patches/ surf | suckless.org software that sucks less
2020-12-27 15:47:00 http://surf.suckless.org/patches/unicode-in-dmenu/ surf | suckless.org software that sucks less
2020-12-27 15:48:02 https://www.youtube.com/ YouTube
2020-12-27 15:48:34 https://gitlab.com/dwt1 Derek Taylor · GitLab
2020-12-27 15:49:11 https://gitlab.com/dwt1 file:///home/dt/.surf/html/homepage.html
2020-12-27 15:49:11 file:///home/dt/.surf/html/homepage.html 
2020-12-27 15:50:59 file:///home/argha/.surf/html/homepage.html Home
2020-12-27 16:03:58 http://suckless.org/ software that sucks less | suckless.org software that sucks less
2020-12-27 16:04:52 http://suckless.org/ software that sucks less | suckless.org software that sucks less
2020-12-27 16:06:12 http://suckless.org/ software that sucks less | suckless.org software that sucks less
2020-12-27 16:08:07 http://suckless.org/ software that sucks less | suckless.org software that sucks less
2020-12-27 16:18:33 https://gitlab.com/SoumyajitTech Soumyajit Sarkar · GitLab
2020-12-27 16:18:49 http://suckless.org/ software that sucks less | suckless.org software that sucks less
2020-12-27 16:19:09 https://www.youtube.com/ YouTube
2020-12-27 16:20:12 https://archlinux.org/news/accessible-installation-medium/ Arch Linux - News: Accessible installation medium
2020-12-27 16:25:45 https://www.youtube.com/watch?v=ucn2KzCDnOY What's "Natural" anyway? - YouTube
2020-12-27 19:37:37 https://www.reddit.com/r/bspwm/comments/kl185j/a_small_graphic_problem/ A small graphic problem : bspwm
2020-12-27 19:50:53 https://www.reddit.com/r/unixporn/comments/kl1yo1/my_bspwmarch_bspwm/ My Bspwmarch [BSPWM] : unixporn
2020-12-27 19:51:18 https://i.redd.it/09wt834wee761.png 09wt834wee761.png 1920×2198 pixels
2020-12-27 20:00:29 https://www.youtube.com/ YouTube
2020-12-27 20:01:53 https://www.reddit.com/r/commandline/comments/kl3p7t/a_very_simple_system_info_tool_for_linux_written/ A very simple system info tool for Linux written in C : commandline
2020-12-28 15:12:44 https://aur.archlinux.org/ AUR (en) - Home
2020-12-28 15:13:21 https://www.youtube.com/ YouTube
2020-12-28 15:15:35 file:///home/argha/.surf/html/homepage.html Home
2020-12-28 18:47:57 file:///home/argha/.surf/html/homepage.html Home
2020-12-28 20:29:46 https://lukesmith.xyz/ Luke Smith's Webpage
2020-12-28 20:37:37 https://lukesmith.xyz/ Luke Smith's Webpage
2020-12-28 20:43:15 https://lbry.tv/ lbry.tv
2020-12-28 20:44:48 https://lbry.tv/ lbry.tv
2020-12-28 20:58:04 https://lukesmith.xyz/ Luke Smith's Webpage
2020-12-28 20:58:10 https://lukesmith.xyz/articles/programs Programs and Equipment
2020-12-28 20:58:47 https://lukesmith.xyz/ Luke Smith's Webpage
2020-12-28 20:58:49 https://lukesmith.xyz/articles/programs Programs and Equipment
2020-12-28 20:58:59 https://lukesmith.xyz/ Luke Smith's Webpage
2020-12-28 20:59:01 https://lukesmith.xyz/articles/programs Programs and Equipment
2020-12-28 20:59:08 https://lukesmith.xyz/ Luke Smith's Webpage
2020-12-28 22:05:33 https://gitlab.archlinux.org/archlinux/archiso/-/tags/v49 v49 · Tags · Arch Linux / archiso · GitLab
2020-12-28 22:06:47 https://wiki.archlinux.org/index.php/TalkingArch Install Arch Linux with accessibility options - ArchWiki
2020-12-28 22:07:27 https://bugs.archlinux.org/task/68312 FS#68312 : [nvidia] nvidia_uvm uses unknown symbols and breaks CUDA and OpenCL on kernel 5.9.x
2020-12-29 10:33:39 https://archlinux.org/news/accessible-installation-medium/ Arch Linux - News: Accessible installation medium
2020-12-29 11:00:53 https://lukesmith.xyz/ Luke Smith's Webpage
2020-12-29 11:02:33 https://lukesmith.xyz/ Luke Smith's Webpage
2020-12-29 11:02:48 https://brave.com/?ref=luk005 Secure, Fast & Private Web Browser with Adblocker | Brave Browser
2020-12-29 11:03:02 https://lbry.tv/ lbry.tv
2020-12-30 10:47:07 https://lukesmith.xyz/ Luke Smith's Webpage
2020-12-30 10:47:18 https://notrelated.xyz/ Not Related! A Big-Braned Podcast
2020-12-30 20:01:16 https://www.reddit.com/r/archlinux/comments/kmzpug/is_arch_good_for_gaming/ Is Arch good for gaming : archlinux
2020-12-31 10:06:08 https://www.reddit.com/r/archlinux/comments/knik9a/is_it_worth_going_through_the_trouble_of_setting/ Is it worth going through the trouble of setting up a firewall? : archlinux
2020-12-31 10:07:44 https://www.millennialwoes.com/millenniyule?y=2020 Millenniyule Schedyule [MW]


.doom.d/config.el
(add-to-list 'load-path "/usr/local/share/emacs/site-lisp/mu4e")

(setq doom-font (font-spec :family "monospace" :size 14 :weight 'semi-light)
       doom-variable-pitch-font (font-spec :family "sans" :size 13))

(setq doom-theme 'doom-dracula)

(setq org-directory "~/org/")

(setq display-line-numbers-type t)
(global-set-key "\C-x\ t" 'toggle-truncate-lines)

(setq browse-url-browser-function 'eww-browse-url)

(setq neo-window-fixed-size nil)

(require 'magit)

(defun prefer-horizontal-split ()
  (set-variable 'split-height-threshold nil t)
  (set-variable 'split-width-threshold 40 t)) ; make this as low as needed
(add-hook 'markdown-mode-hook 'prefer-horizontal-split)

(map!
  (:after dired
    (:map dired-mode-map
     "C-x i" 'peep-dired
     )))
(evil-define-key 'normal peep-dired-mode-map (kbd "j") 'peep-dired-next-file
                                             (kbd "k") 'peep-dired-prev-file)
(add-hook 'peep-dired-hook 'evil-normalize-keymaps)

(custom-set-variables
 '(elfeed-feeds
   (quote
    (("https://www.reddit.com/r/linux.rss" reddit linux)
     ("https://www.gamingonlinux.com/article_rss.php" gaming linux)
     ("https://hackaday.com/blog/feed/" hackaday linux)
     ("https://opensource.com/feed" opensource linux)
     ("https://linux.softpedia.com/backend.xml" softpedia linux)
     ("https://itsfoss.com/feed/" itsfoss linux)
     ("https://www.zdnet.com/topic/linux/rss.xml" zdnet linux)
     ("https://www.phoronix.com/rss.php" phoronix linux)
     ("http://feeds.feedburner.com/d0od" omgubuntu linux)
     ("https://www.computerworld.com/index.rss" computerworld linux)
     ("https://www.networkworld.com/category/linux/index.rss" networkworld linux)
     ("https://www.techrepublic.com/rssfeeds/topic/open-source/" techrepublic linux)
     ("https://betanews.com/feed" betanews linux)
     ("http://lxer.com/module/newswire/headlines.rss" lxer linux)
     ("https://distrowatch.com/news/dwd.xml" distrowatch linux))))
 '(package-selected-packages
   (quote
    (exwm peep-dired nav-flash evil-mu4e elfeed))))
